samples/msal-node-samples/managed-identity/README.md

```diff
@@ -0,0 +1,54 @@
+# Managed Identity for Azure VM Sample
+
+This sample demonstrates how to use [managed identity via the msal-node library](/lib/msal-node/docs/managed-identity.md) to retrieve tokens for a managed identity application running on an Azure VM.
+
+## Note
+
+-   The functionality for this sample is in preview (alpha)
+-   This sample is written in TypeScript and was developed with Node version 18.17.0.
+
+## Setup
+
+In a terminal, navigate to the directory where `package.json` resides. Then type:
+
+```console
+    npm install
+```
+
+## Register
+
+Follow [this guide](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/qs-configure-portal-windows-vm) to setup an Azure VM, as well as add a system assigned and user assigned managed identity to the Azure VM.
+
+## Local Changes
+
+Before running the sample on the Azure VM, the userAssignedClientId value in the managedIdentityIdParams object in index.ts needs to be replaced by the client id of the user assigned managed identity that was created in the previous step:
```

Comment:
The GitHub suggestion feature doesn't work well... My suggestion is to combine Setup, Register, and Local Changes sections into this:



## Virtual machine setup

Follow [this guide](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/qs-configure-portal-windows-vm) to setup an Azure VM, as well as add a system assigned and user assigned managed identity to the Azure VM.

## Project setup

In a terminal, navigate to the directory where `package.json` resides. Then type:

```console
    npm install
```

Before running the sample on the Azure VM, the `userAssignedClientId` in the `managedIdentityIdParams` object in `index.ts` needs to be replaced by the client ID of the user assigned managed identity that was created in the previous section:
```typescript
const managedIdentityIdParams: ManagedIdentityIdParams = {
    userAssignedClientId: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
};
```